Makefile

```diff
@@ -312,18 +315,21 @@ $(BUILD_DIRECTORY)/$(APPLICATION_NAME_ELECTRON)-$(APPLICATION_VERSION_REDHAT).$(
 	$(NPX) build --linux rpm $(ELECTRON_BUILDER_OPTIONS) \
 		--extraMetadata.name=$(APPLICATION_NAME_ELECTRON) \
 		--extraMetadata.version=$(APPLICATION_VERSION_REDHAT) \
+		--extraMetadata.$(PACKAGE_TYPE_PROPERTY)=rpm \
 		$(DISABLE_UPDATES_ELECTRON_BUILDER_OPTIONS)
 
 $(BUILD_DIRECTORY)/$(APPLICATION_NAME_ELECTRON)_$(APPLICATION_VERSION_DEBIAN)_$(TARGET_ARCH_DEBIAN).deb: \
 	| $(BUILD_DIRECTORY)
 	$(NPX) build --linux deb $(ELECTRON_BUILDER_OPTIONS) \
 		--extraMetadata.name=$(APPLICATION_NAME_ELECTRON) \
 		--extraMetadata.version=$(APPLICATION_VERSION_DEBIAN) \
+		--extraMetadata.$(PACKAGE_TYPE_PROPERTY)=deb \
 		$(DISABLE_UPDATES_ELECTRON_BUILDER_OPTIONS)
 
 $(BUILD_DIRECTORY)/$(APPLICATION_NAME_LOWERCASE)-$(APPLICATION_VERSION)-$(TARGET_ARCH_APPIMAGE).AppImage: \
 	| $(BUILD_DIRECTORY)
-	$(NPX) build --linux AppImage $(ELECTRON_BUILDER_OPTIONS)
+	$(NPX) build --linux AppImage $(ELECTRON_BUILDER_OPTIONS) \
+		--extraMetadata.$(PACKAGE_TYPE_PROPERTY)=appimage
```

Comment:
I wonder if it'd be worth casing this as `AppImage` instead of `appimage`, so that in every Makefle rule the packageType exactly matches the target string given to electron-builder?